core/app/models/spree/stock/simple_coordinator.rb

```diff
@@ -23,7 +23,7 @@ def initialize(order, inventory_units = nil)
         @order = order
         @inventory_units = inventory_units || InventoryUnitBuilder.new(order).units
         @splitters = Spree::Config.environment.stock_splitters
-        @stock_locations = Spree::StockLocation.active
+        @stock_locations = Spree::Config.stock_location_sorter_class.new(Spree::StockLocation.active).sort
```

Comment:
I'd use another variable or method `@sorted_stock_lcoations` for that. It should also help backward compatibility if someone extended this class and was using the `@stock_locations` ivar. what do you think?

Reply:
@kennyadsl the problem with that is if someone overrides @stock_locations and we sort in a different variable (let's say @sorted_stock_locations), they also have to override @sorted_stock_locations, otherwise it will contain the result of the sorting of the original @stock_locations variable.

Reply:
Yes, you are right. One option could be creating another `@active_stock_locations` ivar, use that in the whole class instead of `@stock_locations` which could be deprecated with [DeprecatedInstanceVariableProxy](https://apidock.com/rails/ActiveSupport/Deprecation/DeprecatedInstanceVariableProxy). I feel like it's too complex though, maybe a changelog line is enough here. I'd say, let's see what others think before.

Reply:
Talked with @gmacdougall about this. Since the default sorted is returning the same thing as before maybe no one will have unexpected behavior with this, so I'm ok leaving it as is.